packages/server/src/service/trpc.ts

```diff
@@ -18,9 +18,11 @@ interface HostInfo {
 }
 
 const hostInfo = (request: Request): HostInfo => {
-  // https://stackoverflow.com/questions/10849687/express-js-how-to-get-remote-client-address
+  // https://cloud.google.com/appengine/docs/flexible/nodejs/reference/request-headers
   return {
-    ip: request.connection.remoteAddress,
+    ip: process.env.NODE_ENV !== 'development'
```

Comment:
What about `request.headers['x-forwarded-for'] ?? request.connection.remoteAddress`?

In general, we should not make code depend on `process.env.NODE_ENV` -- it's bad coding practice.  It seems that we don't really care if the wrong values are stored in dev as long as the right values are stored in staging and production.

Reply:
Thanks, I'll stop using `NODE_ENV` from now and onward in case like this. Also I'm only storing a value at dev builds to avoid any possible error of not having this value when running things locally.